src/aioswitcher/device/tools.py

```diff
@@ -158,3 +160,25 @@ def convert_str_to_devicetype(device_type: str) -> DeviceType:
     elif device_type == DeviceType.RUNNER_MINI.value:
         return DeviceType.RUNNER_MINI
     return DeviceType.MINI
+
+
+def get_token(username: str, password: str) -> str:
+    """Make API call to get a Token by username and password."""
+    token = ""
+    url = "https://switcher.co.il/GetKey/GetTok/t.php"
+    data = {"username": username, "password": password}
+
+    response = requests.post(url, data=data)
+
+    if response.status_code == 200:
+        print("Request successful")
+        try:
+            json_response = response.json()
+            print("JSON Response:", json_response)
+            token = json_response.get("token", "")
+
+        except ValueError:
+            print("Response content is not valid JSON")
+    else:
+        print("Request failed with status code:", response.status_code)
+    return token
```

Comment:
Let's encapsulate the token with a value class, and return in as a JSON. It will be more robust and won't break if we'll need to modify something response's body.

Reply:
Do you have an example of what you mean?

Reply:
Sure, take a look at the classes extending the the base response in the [api module](https://github.com/TomerFi/aioswitcher/blob/dev/src/aioswitcher/api/messages.py#L146). Basically they encapsulate responses we got from the device.

Reply:
But I'm actually having second thought here, after thinking about it over night.
I'll write my concerns in the PR comments.

Reply:
Take a look now after using the value class.

Reply:
I don't it's needed anymore if we use it only for the script. We don't need this functions in the module itself.

Reply:
No actually. Shay and I on the same point here.
Thia get token logic should be part of this library in order to be able to call it from HA and the other repo code.

Reply:
Are we going to use this function from the HA?

Reply:
Here. Yes.